elodie/tests/filesystem_test.py

```diff
@@ -46,7 +47,13 @@ def test_create_directory_recursive_success():
 
     shutil.rmtree(folder)
 
-def test_create_directory_invalid_permissions():
+
+@mock.patch('elodie.filesystem.os.makedirs')
```

Comment:
This test was failing when run inside the container, because the user is root there and root does have permission to create the given directory. I changed the test to mock makedirs instead, and force it to raise an OSError.